draft-ietf-quic-multipath.md

```diff
@@ -132,18 +132,24 @@ and the actual decision to setup or tear down paths are assumed
 to be handled by the application. But this document does not prevent future extensions from
 defining mechanisms to cope with the remaining scenarios.
 
-Further, this document does not specify detailed algorithms that define
+Further, this document does not specify scheduling algorithms that define
 how multiple, simultaneously open paths are used to send packets.
-As these differ depending on the application requirements,
+As these differ depending on application requirements,
 only some basic implementation guidance is discussed in {{impl-consideration}}.
+This extension can be used with different scheduling algorithms that,
+e.g., can range from support for failover to simulatenous
+use of the aggreate capacity across all active paths.
+There are IETF specifications for path fail-over, but
+there are currently no IETF standards-track specifications for simultaneously
```

Comment:
Can we maybe just remove „standards-track“? Because this is also true for none standards-track specs.

```suggestion
there are currently no IETF specifications for simultaneously
```

Reply:
RFC 6356, RFC8382 and RFC 8699 are all experimental specifications for scheduling, albeit not targeting QUIC, but they are all EXP.

Reply:
These RFCs are all about congestion control but do not specify scheduling.